tests/unit_test/conftest.py

```
@@ -1,27 +1,35 @@
-		name="Test User"
-	)
-	
-	# Set the override
-	app.dependency_overrides[get_current_user] = lambda: override_data
+def mock_user() -> AuthenticatedUserDTO:
+	return AuthenticatedUserDTO(id="user_abc123", email="user@example.com", name="Test User")
+
+
+@pytest.fixture
+def mock_authenticated_user(mock_user):
+	"""
```

Pylint (3.3.7)

[warning] Bad indentation. Found 1 spaces, expected 4

(W0311)